client/flutter/assets/content_bundles/your_questions_answered.fr_FR.yaml

```diff
@@ -0,0 +1,221 @@
+
+        <p>Les personnes âgées, ainsi que celles qui ont d&#39;autres problèmes de santé telles que de l&#39;hypertension artérielle, des problèmes cardiaques ou du diabète, ont davantage de risques de présenter des symptômes graves. </p>
+
+        <p><b>Toute personne qui a de la fièvre, tousse et éprouve des difficultés à respirer doit consulter un médecin.</b></p>
+    -
+      title_html: |
+        Comment le COVID-19 se propage-t-il ?
+      body_html: |
+        <p>🤧 Le COVID-19 se transmet par des personnes porteuses du virus.</p>
+
+        <p>💦 La maladie peut se transmettre d&#39;un individu à l&#39;autre par le biais des gouttelettes respiratoires expulsées par le nez ou par la bouche lorsqu&#39;une personne tousse ou éternue.</p>
+
+        <p>🥄 Ces gouttelettes peuvent se retrouver sur des objets ou des surfaces autour de la personne en question.</p>
+
+        <p>👈 On peut alors contracter le COVID-19 si l&#39;on touche ces objets ou ces surfaces, et si l&#39;on se touche ensuite les yeux, le nez ou la bouche.</p>
+
+        <p>↔️ Il est également possible de contracter la COVID-19 en inhalant des gouttelettes d&#39;une personne malade qui vient de tousser ou d&#39;éternuer.
+        C&#39;est pourquoi il est important de maintenir une distance d&#39;au moins 1 mètre (3 pieds) avec toute personne malade.</p>
+
+        <p>L&#39;OMS évalue actuellement les travaux de recherche en cours sur la manière dont le COVID-19 se propage, et elle continuera à communiquer des résultats actualisés.</p>
+
+        <p><b>Le virus responsable du COVID-19 est-il transmissible par voie aérienne ?</b> Les études menées à ce jour semblent indiquer que le
+        virus responsable du COVID-19 est principalement transmissible par le contact avec des gouttelettes respiratoires, et non par voie aérienne.</p>
+
+        <p><b>Peut-on contracter le COVID-19 au contact d&#39;une personne qui ne présente aucun symptôme ?</b>
+        La maladie se propage principalement par les gouttelettes respiratoires expulsées par les personnes qui toussent.
+        Le risque de contracter le COVID-19 au contact d&#39;une personne qui ne présente aucun symptôme est très faible.
+        Toutefois, de nombreuses personnes atteintes ne présentent que des symptômes modérés.
+        Ceci est particulièrement vrai aux stades initiaux de la maladie.
+        Il est donc possible de contracter le COVID-19 au contact d&#39;une personne qui, par exemple, présente seulement une toux légère mais ne se sent pas souffrante.</p>
+
+        <p>L&#39;OMS évalue actuellement les travaux de recherche en cours sur la période de transmission du COVID-19, et elle continuera à communiquer des résultats actualisés.</p>
+    -
+      title_html: |
+        Puis-je contracter la COVID-19 par contact avec les matières fécales, à partir d&#39;une source animale ou à partir d&#39;un animal domestique ?
```

Comment:
Its "le COVID-19" in that case

Reply:
WHO seems to be using "la COVID-19" - https://www.who.int/fr/emergencies/diseases/novel-coronavirus-2019/advice-for-public/myth-busters

Reply:
They refer to Maladie, which his nom féminin. That's technically correct but also reads strange to me.

Reply:
I think it's a translation error on the WHO website. "Un" coronavirus is masculine, therefore "le" has to be used. I checked with others public health and newspaper website and 'le' is used everywhere.

Reply:
To be reviewed by WHO in RC3